andela/hc-millennium-falcon
# ft(hc/accounts): Set periodic reporting

What does this PR do?
Gives the user the ability to be able to configure periodic reporting

Description of Task to be completed?
The system allows the user to select period of choice when they will be receiving reports about their checks. the periods include either monthly, weekly or daily 

How should this be manually tested?
run ./manage.py runserver.
run ./manage.py sendreports.
configure sendgrid environmental key using export SENDGRID_API_KEY=paste_api_key_here. Then
Create a new check and ping the url. After that, set reporting period and whether to receive reports or not from Accounts settings page. 
**Note** : 
To receive email reports, you must select the option "send me a summary of my checks" on the Account settings page.

Any background context you want to provide?
previously the default setting was that a user only gets notified monthly.

What are the relevant pivotal tracker stories?
[156577918](https://www.pivotaltracker.com/story/show/156577918)
